components/prize_pool/wikis/commons/starcraft_starcraft2/prize_pool_starcraft.lua

```diff
@@ -0,0 +1,371 @@
+		return lpdbData.objectName .. '_' .. prizePoolIndex
+	end
+
+	return lpdbData.objectName
+end
+
+function CustomPrizePool._getMode(opponentType, opponent)
+	if (opponent or {}).isArchon then
+		return 'archon'
+	end
+
+	return OPPONENT_TYPE_TO_MODE[opponentType or '']
+end
+
+function CustomPrizePool._getStatusFromScore(score)
+	return Logic.isNumeric(score) and SCORE_STATUS or score
+end
+
+function CustomPrizePool._appendLpdbTournament()
+	local tournamentName = Variables.varDefault('tournament_name', mw.title.getCurrentTitle().text)
+
+	local extradata = {
+		seriesnumber = _series_number,
+		featured = Variables.varDefault('featured') or 'false'
+	}
+
+	for _, placement in pairs(_tournament_extradata_cache) do
+		Table.mergeInto(
+			extradata,
+			CustomPrizePool._placementToTournamentExtradata(placement)
+		)
+	end
+
+	mw.ext.LiquipediaDB.lpdb_tournament(
+		'tournament_' .. tournamentName,
+		{extradata = Json.stringify(extradata)}
+	)
+end
+
+function CustomPrizePool._placementToTournamentExtradata(entries)
+	if not entries[1] then
+		return {}
+	end
+
+	local prefix = PLACE_TO_KEY_PREFIX[entries[1].placement]
+	if (prefix and #entries > 1) or (not prefix and #entries > 2) then
+		return {}
+	end
+
+	if not prefix then
+		return Table.merge(
+			CustomPrizePool._entryToTournamentExtradata(SEMIFINALS_PREFIX .. 1, entries[1]),
+			CustomPrizePool._entryToTournamentExtradata(SEMIFINALS_PREFIX .. 2, entries[2])
+		)
+	end
+
+	return CustomPrizePool._entryToTournamentExtradata(prefix, entries[1])
+end
+
+function CustomPrizePool._entryToTournamentExtradata(prefix, entry)
+	local opponent = StarcraftOpponent.fromLpdbStruct{
+		opponentplayers = entry.players or {},
+		opponenttype = entry.opponenttype,
+		opponentname = entry.participantlink,
+		template = entry.participanttemplate,
+	}
+
+	local function toLink(player)
+		return player.pageName
+			and player.pageName .. '|' .. player.displayName
+			or player.displayName
+	end
+
+	if opponent.type == StarcraftOpponent.solo then
+		return {
+			[prefix] = toLink(opponent.players[1]),
+			[prefix .. 'flag'] = opponent.players[1].flag,
+			[prefix .. 'link'] = opponent.players[1].pageName,
+			[prefix .. 'race'] = opponent.players[1].race,
+		}
+	elseif StarcraftOpponent.typeIsParty(opponent.type) then
+		local extradata = {}
+		if opponent.isArchon then
+			extradata[prefix .. 'race'] = opponent.players[1].race
+		end
+		for playerIndex, player in ipairs(opponent.players) do
+			extradata[prefix .. 'p' .. playerIndex] = toLink(player)
+			extradata[prefix .. 'flagp' .. playerIndex] = player.flag
+			extradata[prefix .. 'linkp' .. playerIndex] = player.pageName
+			extradata[prefix .. 'racep' .. playerIndex]
+				= not opponent.isArchon and player.flag or nil
+		end
+		return extradata
+	elseif opponent.type == StarcraftOpponent.team then
+		return {[prefix] = opponent.name}
+	end
+
+	return {[prefix] = TBD}
+end
+
+function CustomSmwInjector:adjust(smwEntry, lpdbEntry)
```

Comment:
I'd be fine removing anything that's excessive from SMW

Reply:
what would you consider as excessive?
from the usage on the wiki standpoint everything is
from a consumer standpoint probably the `additionalCommonProps`, possibly lastVs?

Reply:
I leave that up to you :)